product-service-ts/handlers/products-mock.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "id": "7567ec4b-b10c-48c5-9345-fc73c48a80aa",
+        "description": "Short Product Description1",
```

Comment:
Should you also store productImageUrl? I see right now that all your images are the same

Reply:
Thanks, done.

P.S.
Added imageUrl.